`GmlExporterTest.testGraphicsSection()` Failing on Windows

### Environment

 * JGraphT version: 4ec044c
 * Java version/platform: OpenJDK 17.0.9 / Windows 11 23H2 (x64)

### Issue

jgrapht-io tests fail to run on Windows

### Steps to reproduce (small coding example)

1. Checkout repository (on a Windows machine)
2. `cd` to the local repository
3. Run `mvn test`

### Expected behavior

All tests run green on Windows

### Actual behavior

```text
[ERROR] Tests run: 13, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.129 s <<< FAILURE! -- in org.jgrapht.nio.gml.GmlExporterTest
[ERROR] org.jgrapht.nio.gml.GmlExporterTest.testGraphicsSection -- Time elapsed: 0.012 s <<< FAILURE!
org.junit.ComparisonFailure:
expected:<...
                id 1
                graphics[
                [
                        fill "#FF0000"
                ]
        ]
        node
        [
                id 2
]               graphics
                [
                        fill "#FF00...> but was:<...
                id 1
                graphics[
                [
                        fill "#FF0000"
                ]
        ]
        node
        [
                id 2
                graphics]
                [
                        fill "#FF00...>
        at org.junit.Assert.assertEquals(Assert.java:117)
        (...)
```

### Other information

- Found while working on #1163 
- This does not occur on Linux (or else GHA would have been yelling out errors already)
- The outputs (expected / actual) look alike. If I were to guess the cause of the error, I would say CRLF / LF inconsistency happening somewhere.

--- a/jgrapht-io/src/main/java/org/jgrapht/nio/gml/GmlExporter.java
+++ b/jgrapht-io/src/main/java/org/jgrapht/nio/gml/GmlExporter.java
@@ -283,7 +283,7 @@ public class GmlExporter<V, E>
             }
             if (exportCustomNodeGraphicsAttributes) {
                 getVertexGraphicsAttributes(from).ifPresent(graphicsAttributes -> {
-                    out.println(TAB2 + "graphics\n" + TAB2 + "[");
+                    out.println(TAB2 + "graphics" + System.lineSeparator() + TAB2 + "[");
                     graphicsAttributes.entrySet().stream().forEach(e -> {
                         String customAttributeKey = e.getKey();
                         Attribute customAttributeValue = e.getValue();

